eng/ApiListing.targets

```diff
@@ -0,0 +1,57 @@
+<Project>
```

Comment:
Is there a reason the default targets coming from the package will not work? https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.GenAPI/build/Microsoft.DotNet.GenAPI.targets

Reply:
Yes, I wanted more checks that are specific to Azure SDKs and I had experience of being broken by changes in those targets.

Reply:
There should be enough properties to override the specific things we want for us and if we can do that with a couple property changes we should do it as opposed to duplicating it.

Reply:
Done